.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
```

Comment:
What might be good here to also state is something like this at the beginning of the issue:

```mkd
## Before Filing a Report

- [ ] Is my feature request in-line with the Javis philosophy?
- [ ] Did I search the Javis GitHub to see if this feature has already been requested?
```